cve_bin_tool/checkers/chrome.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2022 Orange
```

Comment:
We probably need a little updating here, heres the default header we use : https://github.com/intel/cve-bin-tool/blob/main/spdx_header.txt
you can switch out Intel with your name if you want.

Reply:
Will make the necessary changes